Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/EffectData.xml

```diff
@@ -22909,4 +22909,90 @@
         <EffectArray value="AP_MengskVeteranMaxedTrigger"/>
         <EffectArray value="AP_ThorMengskWeaponChange"/>
     </CEffectSet>
+    <CEffectDamage id="AP_BanelingLaunchExplodeDamageSmall" parent="DU_WEAP">
+        <AINotifyFlags index="HurtEnemy" value="1"/>
+        <ImpactLocation Value="TargetUnit"/>
+        <AttributeBonus index="Structure" value="30"/>
+        <Kind value="Splash"/>
+        <Amount value="30"/>
+        <Death value="Disintegrate"/>
+        <AreaArray Radius="2.2" Fraction="1"/>
+        <ExcludeArray Value="Target"/>
+        <SearchFilters value="Air;Player,Ally,Neutral,Missile,Item,Stasis,Dead,Hidden,Invulnerable"/>
+    </CEffectDamage>

```

Comment:
if it's an active ability, the damage should be spell, also should ignore armor (remove the parent)

Reply:
Removed the parent and the damage type line (should default to spell damage and 0 armor reduction now)